packages/fluentui/styles/src/types.ts

```diff
@@ -32,8 +32,9 @@ export interface AnimationName<P = Record<string, any>> {
   params?: P;
 }
 
-type CSSProperties = Omit<CSS.Properties<string | number>, 'animationName'> & {
+type CSSProperties = Omit<CSS.Properties<string | number>, 'animationName' | 'position'> & {
   animationName?: AnimationName | AnimationKeyFrame | string | 'none';
+  position?: string;
```

Comment:
We should not relax `position` to `string` as this type is used across all style functions in N*.

---

- Restore this change
- 
```suggestion
type CSSProperties = Omit<CSS.Properties<string | number>, 'animationName'> & {
  animationName?: AnimationName | AnimationKeyFrame | string | 'none';
```

- Update `packages/fluentui/react-proptypes/src/index.ts`

```diff
// Keep in sync with packages/react/src/themes/types.ts
export const design = PropTypes.shape({
-  position: PropTypes.string,
-  display: PropTypes.string,
+  position: PropTypes.string as PropTypes.Requireable<any>,
+  display: PropTypes.string as PropTypes.Requireable<any>,
```

Reply:
Fixed